.github/workflows/ateam.yml

```diff
@@ -19,14 +19,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Create SSH key
+      - name: Create yum repo
         run: |
-          mkdir -p -m 700 ~/.ssh/
-          echo "${{ secrets.SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa
-          chmod 600 ~/.ssh/id_rsa
-        shell: bash
-
-      - name: Image build
-        run: |
-            tmt run -vvv --all provision --how connect --guest ${{ secrets.HOST }} --key ~/.ssh/id_rsa
-        shell: bash
+          docker build -f ci/create_yum_repo/Dockerfile -t create_yum_repo .
+          docker run -it create_yum_repo
```

Comment:
Maybe it'd be easier to use centos, or fedora, container for the jobs and move the logic from the Dockerfile to here.
Here we set what container we use:
https://github.com/osbuild/automotive-ci/blob/main/.github/workflows/ateam.yml#L17

As you're not running the others jobs for your tests, you should be able to change the container. Also, I think you can use different containers for different jobs:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainerimage

Reply:
I've take your advice. Thanks.